python/ambassador/config/config.py

```diff
@@ -218,19 +221,10 @@ def as_dict(self) -> Dict[str, Any]:
     def as_json(self):
         return json.dumps(self.as_dict(), sort_keys=True, indent=4)
 
-    def ambassador_id_required(self, resource_kind: str) -> bool:
-        if resource_kind in self.KnativeResources:
-            return False
-
-        return True
-
     # Often good_ambassador_id will be passed an ACResource, but sometimes
     # just a plain old dict.
     def good_ambassador_id(self, resource: dict):
-        resource_kind = resource.get('kind', '').lower()
-        if not self.ambassador_id_required(resource_kind):
```

Comment:
I removed this check because it seemed like it prevented this method from working with regular Ambassador resources. But I can add it back if that behavior is intentional.